third_party/move/tools/move-formatter/tests/aptos_framework_case/coin.spec.fmt.move

```diff
@@ -0,0 +1,613 @@
+        /// [high-level-req-1.2]
+        aborts_if type_info::type_of<CoinType>().account_address != account_addr;
+        /// [high-level-req-2]
+        aborts_if exists<CoinInfo<CoinType>>(account_addr);
+        aborts_if string::length(name) > MAX_COIN_NAME_LENGTH;
+        aborts_if string::length(symbol) > MAX_COIN_SYMBOL_LENGTH;
+    }
+
+    // `account` must be `@aptos_framework`.
+    spec initialize_with_parallelizable_supply<CoinType>(
+        account: &signer,
+        name: string::String,
+        symbol: string::String,
+        decimals: u8,
+        monitor_supply: bool,
+    ): (
+        BurnCapability<CoinType>,
+        FreezeCapability<CoinType>,
+        MintCapability<CoinType>
+    ) {
+        use aptos_framework::aggregator_factory;
+        let addr = signer::address_of(account);
+        aborts_if addr != @aptos_framework;
+        aborts_if monitor_supply && !exists<
+            aggregator_factory::AggregatorFactory
+        >(@aptos_framework);
+        include InitializeInternalSchema<CoinType> {
+            name: name.bytes,
+            symbol: symbol.bytes
+        };
+        ensures exists<CoinInfo<CoinType>>(addr);
+    }
+
+    /// Make sure `name` and `symbol` are legal length.
+    /// Only the creator of `CoinType` can initialize.
+    spec schema InitializeInternalSchema<CoinType> {
+        account: signer;
+        name: vector<u8>;
+        symbol: vector<u8>;
+        let account_addr = signer::address_of(account);
+        let coin_address = type_info::type_of<CoinType>().account_address;
+        aborts_if coin_address != account_addr;
+        aborts_if exists<CoinInfo<CoinType>>(account_addr);
+        aborts_if len(name) > MAX_COIN_NAME_LENGTH;
+        aborts_if len(symbol) > MAX_COIN_SYMBOL_LENGTH;
+    }
+
+    spec initialize_internal<CoinType>(
+        account: &signer,
+        name: string::String,
+        symbol: string::String,
+        decimals: u8,
+        monitor_supply: bool,
+        parallelizable: bool,
+    ): (
+        BurnCapability<CoinType>,
+        FreezeCapability<CoinType>,
+        MintCapability<CoinType>
+    ) {
+        include InitializeInternalSchema<CoinType> {
+            name: name.bytes,
+            symbol: symbol.bytes
+        };
+        let account_addr = signer::address_of(account);
+        let post coin_info = global<CoinInfo<CoinType>>(account_addr);
+        let post supply = option::spec_borrow(coin_info.supply);
+        let post value = optional_aggregator::optional_aggregator_value(supply);
+        let post limit = optional_aggregator::optional_aggregator_limit(supply);
+        modifies global<CoinInfo<CoinType>>(account_addr);
+        aborts_if monitor_supply && parallelizable && !exists<
+            aggregator_factory::AggregatorFactory
+        >(@aptos_framework);
+        /// [managed_coin::high-level-req-2]
+        ensures exists<CoinInfo<CoinType>>(account_addr) && coin_info.name == name && coin_info
```

Comment:
The line-breaking behavior here is consistent with rustfmt. For example, if the line ends with `aa.bb`, it will be split into lines as `aa\n[$indent].bb`